flashbots/flashbots-writings-website
# SEO: Remove support for trailing slashes

Configure both Docusaurs and Vercel to not immediately resolve pages with a `/` at the end. Instead redirect - HTTP301 - to the URL without a slash.

### To Test
1. Go to any post in the [preview website](https://flashbots-writings-website-git-gkoscky-disable-7c2c00-flashbots.vercel.app), copy the URL
2. Visit the [redirect checker](https://www.whatsmydns.net/redirect-checker), paste the copied URL, and click "Check Redirects"
3. The resulting table should have one row, containing the pasted URL with the "Code" 200 - OK
    | URL | Code | Description |
    | --- | ---- | ----------- |
    | `pasted-url-here` | 200 | OK |

4. Add a `/` at the end of the URL in the input field and click "Check Redirects"
5. The resulting table should have two rows, the first one with a "Code" 308 - Permanent Redirect, followed by a "200" - OK
    | URL | Code | Description |
    | --- | ---- | ----------- |
    | `pasted-url-here` | 308 | Permanent redirect |
    | `pasted-url-here` | 200 | OK |

---
Task: [Page canonicalization issues](https://docs.google.com/presentation/d/1zRznWdOHz9ijfv-TOfkGSjqHv5gAFSETFtJ-e1d6wvI/edit#slide=id.g1365e349eeb_1_2)
